src/panel-rooms.tsx

```diff
@@ -166,9 +166,14 @@ class RoomsPanel extends PSRoomPanel {
 	}
 	renderRoomList(title: string, rooms?: RoomInfo[]) {
 		if (!rooms || !rooms.length) return null;
+		const sortedRooms = rooms.sort((a, b) => {
+			if (!b.userCount || !a.userCount) return 0;
```

Comment:
?

Reply:
Defaulting to `0` seems a better plan than making a comparison function that doesn't uphold global comparison guarantees.

Reply:
```suggestion
```

Reply:
Because `userCount` can be a false-y value as per the definition https://github.com/Zarel/Pokemon-Showdown-Client/blob/master/src/panel-mainmenu.tsx#L8

You also can't return undefined, so I returned zero because that's the number that most makes sense in this case.